requirements.txt

```
@@ -1,5 +1,5 @@
 --find-links https://prototyp3-dev.github.io/pip-wheels-riscv/wheels/
-cartesapp @ git+https://github.com/prototyp3-dev/cartesapp@1910c50e6cb0adc44d1b03b59fb9d333cfc17fbb
+cartesapp @ git+https://github.com/prototyp3-dev/cartesapp@48ac2b24e7b710bac8ac9948135f394ca943ce11
```

Maybe it is better to reference a branch.

```suggestion
cartesapp @ git+https://github.com/prototyp3-dev/cartesapp@main
```

---

We should fix a version that works for the current code, otherwise it could break uncontrollably.